### DISCUSSION TITLE
Aws s3 support for the GraphRAG

### DISCUSSION THREAD

**Discussion by algosmith** [microsoft/graphrag#1307] [Ideas]
I am working with GraphRag I have added the support for the AWS s3 and other AWS products. Now it take the input from AWS s3 and save the output files there.

Can we integrate that feature here in code ? or is it exclusive to the Microsoft products?

**Comment by shahzebali42** [comment]
Can you please share steps and info about what awa services are you using and how? I am actually looking for the same.

**Comment by jgbradley1** [comment]
I can't give a definitive answer to this question right now.

What I can say is we're investigating a few options to promote community contributions *soon* within the repository but without taking a hard dependency on them.

Vector stores is a great example - many options exist but we're unable to maintain/support every vector database out there. We've had some fairly large code refactors recently to clean things up. Setting up a process to support community contributions (beyond addressing bugs and new features) is one of our goals but doing so before a v1 release is premature and could lead to more frustration due to breaking changes in certain code interfaces.
